CLAUDE.md

```
@@ -0,0 +1,43 @@
+make acceptance-persist      # Run tests and keep the KinD cluster alive for debugging
+make acceptance-kubeconfig   # Export KUBECONFIG for a persisted cluster: eval $(make acceptance-kubeconfig)
+```
+
+Run a specific feature:
+```bash
+cd acceptance && go test -count=1 -v -timeout 30m ./... -run TestFeatures/feature_name
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)